server/main.lua

```diff
@@ -7,6 +7,7 @@ local FingerDrops = {}
 local Objects = {}
 local QBCore = exports['qb-core']:GetCoreObject()
 local updatingCops = false
+local exploit_distance_check = 2.5 -- If you are getting exploit bans, increase this number
```

Comment:
This should be as a config variable.  If you want it user-editable, please place it there.